src/Stripe.net/Services/Issuing/Disputes/DisputeCreateOptions.cs

```diff
@@ -3,17 +3,40 @@ namespace Stripe.Issuing
     using System.Collections.Generic;
     using Newtonsoft.Json;
 
-    public class DisputeCreateOptions : CardholderSharedOptions
```

Comment:
ouch this was bad :(
Technically it means we're removing properties that stripe-dotnet supported (the ones in `CardholderSharedOptions`) even though they would never have worked. So it's technically a breaking change because some code might not compile anymore, though that code would never have worked so I'm fine as a minor, but flagging for learning!)

Reply:
yeah this seemed odd to me when I encountered it, but thought it was mostly a naming thing